### ISSUE TITLE
--onion does not work in docker

### ISSUE THREAD

**Issue by JavierRSobrino** [shesek/spark-wallet#53]
Tor bundle is not correctly installed in the docker when you call it with `--onion`. It seems that `xz` program is missing.

```
Downloading Tor Bundle from https://dist.torproject.org/torbrowser/8.0.4/tor-browser-linux64-8.0.4_en-US.tar.xz...
Unpacking Tor Bundle into /data/spark/tor/tor-installation/node_modules/granax/bin...
tar (child): xz: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
Failed to unpack bundle
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! granax@3.1.3 postinstall: `node script/download-tbb.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the granax@3.1.3 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-01-10T15_13_55_003Z-debug.log
unhandledRejection, stopping process
Error: Tor installation failed, exited with code 1
    at ChildProcess.<anonymous> (/opt/spark/dist/transport/hsv3-dep/index.js:54:55)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
Error: Tor installation failed, exited with code 1
    at ChildProcess.<anonymous> (/opt/spark/dist/transport/hsv3-dep/index.js:54:55)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
```

**Comment by shesek** [comment]
It looks like the standalone image was missing the `xz-utils` dependency. Can you confirm this is what you were using?

I added the missing dependency in 0da0244, will make a new version release in the coming days.

**Comment by JavierRSobrino** [comment]
I was using `shesek/spark-wallet:standalone`.

**Comment by shesek** [comment]
Released in v0.2.3.
